go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -153,7 +153,7 @@ func (oc *Controller) updateEgressFirewall(oldEgressFirewall, newEgressFirewall
 		[]egressfirewallapi.EgressFirewallPort{},
 	)
 
-	err = createLogicalRouterPolicy(priority, match, "drop", newEgressFirewall.Namespace+"-blockAll")
+	err = createEgressFirewallRules(priority, match, "drop", newEgressFirewall.Namespace+"-blockAll")
```

Comment:
same change request as the deleteEgressFirewallRules() function

Reply:
fixed